src/index.js

```diff
@@ -213,6 +220,7 @@ export {
 	HomeIcon,
 	Icon,
 	ImageIcon,
+	InfiniteSlidePanel,
```

Comment:
It seems like it's case by case -- we definitely expose several private components in `anx-react`.